source/plugins/go/src/extension/interfaces.go

```diff
@@ -13,7 +13,7 @@ type AgentTaggedDataResponse struct {
 // TaggedData structure for respone
 type TaggedData struct {
 	SchemaVersion           int                         `json:"schemaVersion"`
-	Version                 int                         `json:"version"`
+	Version                 string                      `json:"version"`

```

Comment:
why the data type of this filed changed from int to string? Did we verify this change works on both Linux and Windows?

Reply:
The version was: 2.0.0. This was giving an error on both Windows and Linux with the datatype as string. It probably worked before because we used to convert to make multiple conversions from bytes to string and again to bytes. But have removed any unnecessary conversions so it is better to have it as a string. Works on both Linux and Windows